qt/python/mantidqt/widgets/workspacedisplay/table/model.py

```diff
@@ -102,21 +103,36 @@ def get_number_of_columns(self):
     def get_column_header(self, index):
         return self.get_column_headers()[index]
 
+    def is_editable_column(self, icol):
+        if self.is_peaks_workspace():
+            return self.ws.getColumnNames()[icol] in self.EDITABLE_COLUMN_NAMES
+        else:
+            return False
```

Comment:
I think this is supposed to return True if the workspace isn't a peaks workspace.

In which case it can be simplified to:
```suggestion
        return not self.is_peaks_workspace() or self.ws.getColumnNames()[icol] in self.EDITABLE_COLUMN_NAMES
```

Reply:
Good spot, I put False instead of True on L110 by mistake! Would you prefer it to be all on one line, I thought that might lack readability?

Reply:
I tend to prefer a single return statement where possible but I see your point about readability. I'll let you decide.